src/pipeline/compatibility.rs

```diff
@@ -355,7 +355,7 @@ unsafe fn create_program(
         match (version.major, version.minor, version.is_embedded) {
             // OpenGL 3.0+
             (3, 0 | 1 | 2, false) => (
-                format!("#version 1{}0", version.minor + 3),
+                format!("#version 1{}0\n#extension GL_ARB_explicit_attrib_location : enable", version.minor + 3),
```

Comment:
You shouldn't need to add this here, since `compatibility::Pipeline` doesn't use the "layout qualifiers" that are causing the `panic`.

Reply:
Fixed